parent/pom.xml

```diff
@@ -28,7 +28,8 @@
   <properties>
     <!-- yyyy-MM-dd'T'HH:mm:ss'Z' -->
     <project.build.outputTimestamp>2023-09-03T16:20:19Z</project.build.outputTimestamp>
-    <latest.1.version>1.7.36</latest.1.version>
+    <!-- package version being backwards compatible with SLF4J 1.x OSGi consumers -->
+    <backwards.compatible.version>1.8.0</backwards.compatible.version>
```

Comment:
Can you explain why you choose 1.8.0?
Did you do it because of the SemVer rules to increase the minor version if API is added?
@ceki could it happen that there will be a 1.8 version of slf4j one day?
If yes, I suggest to use something very high, e.g. 1.99 so that the package from slf4j-api 2 is always higher tan a package exported from a potential future slf4j-1.X release.

Reply:
Sure, I incremented the minor version against the 1.7.36 release because that is what https://docs.osgi.org/whitepaper/semantic-versioning/ recommends. In this case  the following applies:

> minor — API consumers are compatible with exporters that have the same major number and an equal or higher minor version. API providers are compatible with exporters that have the same major and minor version number. For example, 1.2 is backward compatible with 1.1 for consumers but for providers it is incompatible. Consumers should therefore import [1.2,2) and providers should import [1.2,1.3).

In this case only new classes and new methods have been added but none removed or modified in a backwards incompatible way (that is what baselining proofs as well)

> could it happen that there will be a 1.8 version of slf4j one day?

It should really be 1.7.37 (or other 1.7 versions only increasing the micro version) as according to https://www.slf4j.org/download.html

> The older stable SLF4J version is 1.7.36. It is no longer actively developed.

To me this means, if at all bugfix release only with no changed API signatures!